src/SmartComponents/FormComponents/DataListLayout.test.js

```diff
@@ -2,18 +2,23 @@ import React from 'react';
 import { mount } from 'enzyme';
 import toJson from 'enzyme-to-json';
 import DataListLayout from './DataListLayout';
-
+import { Form } from '@data-driven-forms/react-form-renderer';
 describe('DataListLayout checkbox tests', () => {
 
     it('should render correctly', () => {
         const wrapper = mount(
-            <DataListLayout
-                label="test label"
-                sections={ [{ label: 'test', fields: [{ fields: []}]}] }
-                formOptions={ {
-                    renderForm: () => 'test'
-                } }
-            />
+            <Form onSubmit={ () => undefined }>
+                {(props) => (<DataListLayout
+                    label="test label"
+                    sections={ [{ label: 'test', fields: [{ fields: []}]}] }
+                    formOptions={ {
+                        renderForm: () => 'test'
+                    } }
+                    clearedValue
```

Comment:
this will set `clearedValue` to true. Is that something you want?

Reply:
Without this the tests were failing :(

Reply:
so the renderer does actually consume clearedValue, my point is if this is supposed to be `true` or some other value like `null` or some string. It has to be there if you are using it.

Reply:
I don't use it anywhere, but without it being assigned anything the tests were failing. The value does not matter AFAIK.

Reply:
ok